crypto/cversion.c

```diff
@@ -43,6 +44,30 @@ const char *OPENSSL_version_build_metadata(void)
     return OPENSSL_VERSION_BUILD_METADATA_STR;
 }
 
+static char ossl_cpu_info_str[80];
+static CRYPTO_ONCE init = CRYPTO_ONCE_STATIC_INIT;
+
+DEFINE_RUN_ONCE_STATIC(ossl_init_cpu_info_str)
+{
+#if     defined(__i386)   || defined(__i386__)   || defined(_M_IX86) || \
```

Comment:
no getenv needed?

Reply:
no this I found out I can't guess what happens when no environment variable is used,
especially when I get just a user reporting the bad performance of CPU X

Reply:
Of course OPENSSL_cpuid_setup is supposed be already executed.

Reply:
And we have 128 bits and each is relevant.

Reply:
So the fact that the CPU-setup is already called answers my point.  But maybe including the env var in the string will be useful?

Reply:
Don't know, hope that is not so common to be used.